app/models/induction_coordinator_profile.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
-class InductionCoordinatorProfile < ApplicationRecord
+class InductionCoordinatorProfile < BaseProfile
   belongs_to :user
   has_and_belongs_to_many :schools
+
+  include Discard::Model
+  default_scope -> { kept }
```

Comment:
Is there any reason these can't go on the base profile?

Reply:
Done